sksurv/tree/tree.py

```diff
@@ -225,8 +231,9 @@ def fit(self, X, y, sample_weight=None, check_input=True):
         params = self._check_params(n_samples)
 
         self.n_outputs_ = self.event_times_.shape[0]
-        # one "class" for CHF, one for survival function
-        self.n_classes_ = np.ones(self.n_outputs_, dtype=np.intp) * 2
+        # one "class" for CHF, one for survival function, one as placeholder
+        j_delta = 3
+        self.n_classes_ = np.ones(self.n_outputs_, dtype=np.intp) * j_delta
```

Comment:
This is an interim development step change i.e. my thinking is along the following lines:
* step 1: change computation code to compute a third value that matches the first value (but don't use the third value as yet)
* step 2: change using code to use the third value instead of the first value (and still do sum-ing)
* step 3: change code to output differently shaped values (in low memory mode)